articles/network-types-topologies/index.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/network-types-topologies/
+title: Introduction to Network Types and Network Topologies
+description:
+author: kanishkvardhan-a-n
+date: 2020-10-16T00:00:00-05:00
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/network-types-topologies/hero.jpg
+    alt:  example image
+---
+Computers are used in almost every aspect of life. They are easy to work with, can handle multiple tasks, and output accurate results. This makes agencies, enterprises, businesses, and workplaces dependent on them. Yet, the fact that they can also store and share data with other computers makes them more useful.
+<!--more-->
+This enables people to access them from anywhere around the world (securely of course). For this to work, computers are interlinked with one another. This interconnection between two or more computers forms a [computer network](https://en.wikipedia.org/wiki/Computer_network).
+
+They are categorized based on their:
+
+-  Geographical spread of network strength.
+-  Type of usage and applications.
+-  Mode of transmission and some other factors.
+
+A typical networking setup requires two types of devices: **Servers and Workstations.**
+Servers are the computers responsible for powering the network. Workstations are the devices that make up the network. Some of those devices could be printers, laptops, smartphones, etc. Data transfer and resource sharing is done from the servers. The shared materials are downloaded on the workstations in the form of data packets. While some can be used across countries, others can barely be used within a building. Here are some of the most popular types of networks in use.

```

Comment:
This sentence left me wondering as to Why?
"While some can be used across countries, others can barely be used within a building."
Why is it that some packets can travel the world and other can barely travel within buildings? Is it due to the network topology - if so a short sentence clarifying this would help the audience read along.

Reply:
Right. So I've placed a new topic on network transmission and how they affect the range/extent of the network.
I would like to provide an external link to the "differences between bridges and repeaters". But I feel that the readers should be familiar with the OSI model and whatabouts of data link layer and physical layer. What's your call on this @hectorkambow ?